main.py

```diff
@@ -4,7 +4,11 @@
 
 @app.route("/")
 def hello_patient0():
-    return "Hello Patient 0 you fkin noob. Dont Try HArd"
+    return "Hello Patient 0 you fkin noob. Dont Try HArd fk u"
+@app.route("/faraz")
+def hello_patient01():
+    return "Hello Patient 0 you fkin noob. Dont Try HArd fk u"
+
 
 if __name__ == '__main__':
     app.run("0.0.0.0",port=8000)
```

Comment:
Use variables in function calls instead of values.
A constants.py file might help in organizing a lot of variables that can be used frequently.